src/scripts/firebase_upload.py

```diff
@@ -1,6 +1,7 @@
 import csv
 import os
 import re
+import logging
```

Comment:
order built-in modules alphabetically

Reply:
also, where is this used? pls remove or change prints to logging

Reply:
meant to change prints to log, will do soon

Reply:
ordering done